docs/administration.md

```diff
@@ -25,31 +26,37 @@ Options available to any installation of paperless:
 
 Options available to docker installations:
 
-- Backup the docker volumes. These usually reside within
-  `/var/lib/docker/volumes` on the host and you need to be root in
-  order to access them.
+-   Backup the docker volumes. These usually reside within
+    `/var/lib/docker/volumes` on the host and you need to be root in
+    order to access them.
 
-  Paperless uses 4 volumes:
+    Paperless uses 4 volumes:
```

Comment:
We should probably also note that as of #132 paperless also uses a `paperless_redisdata` volume for the broker (although to be fair, I don't think much important data is stored there)

Reply:
I dont think theres any reason to back that up, its all ephemeral data that gets recreated